types/sharp/index.d.ts

```diff
@@ -1048,6 +1048,10 @@ declare namespace sharp {
         cropOffsetLeft?: number;
         /** Only defined when using a crop strategy */
         cropOffsetTop?: number;
+        /** Only defined when using a trim method */
+        trimOffsetLeft?: number;
+        /** Only defined when using a trim method */
+        trimOffsetTop?: number;
```

Comment:
I can’t find these in the actual sharp source code. Can you link to another source for this info? I don’t think a doc comment elsewhere in the file is a sufficient authority.

Reply:
Alternatively, @Fdebijl, when you said you tested locally, did you mean that you confirmed these properties actually exist at runtime with the latest version of sharp? Or did you just mean that you tested that the type definitions do what they intend? As long as someone can definitively prove that this is right, I’m happy.

Reply:
@andrewbranch I've verified the properties actually exist with sharp 0.28.0
![image](https://user-images.githubusercontent.com/18357629/113106528-277d1c80-9203-11eb-9f72-02d70bfa3c7f.png)

Reply:
Perfect, thank you!